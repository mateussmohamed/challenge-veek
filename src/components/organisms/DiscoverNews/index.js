import React from 'react'
import styled from 'styled-components'

import { Grid, Row, Col } from 'react-styled-flexboxgrid'

import { Button } from 'atoms'
import { Card } from 'molecules'

const Container = styled(Grid)`
  margin: 250px auto;
`

const DiscoverNews = () => (
  <Container>
    <Row>
      <Col xs={12} sm={6} md={4}>
        <Card />
      </Col>
      <Col xs={12} sm={6} md={4}>
        <Card />
      </Col>
      <Col xs={12} sm={6} md={4}>
        <Card />
      </Col>
      <Col xs={12} sm={6} md={4}>
        <Card />
      </Col>
      <Col xs={12} sm={6} md={4}>
        <Card />
      </Col>
      <Col xs={12} sm={6} md={4}>
        <Card />
      </Col>
    </Row>
    <Row center="xs">
      <Col>
        <Button>Discover News</Button>
      </Col>
    </Row>
  </Container>
)

export default DiscoverNews
