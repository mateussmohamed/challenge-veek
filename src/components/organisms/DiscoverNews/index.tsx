import React from 'react'
import styled from 'styled-components'

import { Grid, Row, Col } from 'react-styled-flexboxgrid'

import { Button } from '../../atoms'
import { Card } from '../../molecules'

const DiscoverNewsContainer = styled(Grid)`
  height: 100%;
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  padding-bottom: 20px;
  padding-top: 50px;
`

export const DiscoverNews = () => (
  <DiscoverNewsContainer>
    <Row>
      <Col xs={12} sm={6} md={4}>
        <Card />
      </Col>
      <Col xs={12} sm={6} md={4}>
        <Card />
      </Col>
      <Col xs={12} sm={6} md={4}>
        <Card />
      </Col>
      <Col xs={12} sm={6} md={4}>
        <Card />
      </Col>
      <Col xs={12} sm={6} md={4}>
        <Card />
      </Col>
      <Col xs={12} sm={6} md={4}>
        <Card />
      </Col>
    </Row>
    <Row center="xs">
      <Col>
        <Button>Discover News</Button>
      </Col>
    </Row>
  </DiscoverNewsContainer>
)
