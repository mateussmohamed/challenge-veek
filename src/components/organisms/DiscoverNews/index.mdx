---
name: DiscoverNews
menu: Organisms
---

import { Playground, PropsTable } from 'docz'
import DiscoverNews from './index'

# DiscoverNews
<PropsTable of={DiscoverNews} />

<Playground>
  <DiscoverNews />
</Playground>
