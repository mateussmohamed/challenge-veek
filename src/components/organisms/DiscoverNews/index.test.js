import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import DiscoverNews from './index'

describe('[<DiscoverNews />]', () => {
  const wrap = (props = {}) => shallow(<DiscoverNews {...props} />)

  test('matches to snapshot', () => {
    const wrapper = wrap()

    expect(wrapper).toMatchSnapshot()
  })
})
