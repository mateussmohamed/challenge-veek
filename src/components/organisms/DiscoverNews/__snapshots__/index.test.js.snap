// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<DiscoverNews />] matches to snapshot 1`] = `
<DiscoverNews__DiscoverNewsContainer>
  <Row>
    <Col
      md={4}
      sm={6}
      xs={12}
    >
      <Card />
    </Col>
    <Col
      md={4}
      sm={6}
      xs={12}
    >
      <Card />
    </Col>
    <Col
      md={4}
      sm={6}
      xs={12}
    >
      <Card />
    </Col>
    <Col
      md={4}
      sm={6}
      xs={12}
    >
      <Card />
    </Col>
    <Col
      md={4}
      sm={6}
      xs={12}
    >
      <Card />
    </Col>
    <Col
      md={4}
      sm={6}
      xs={12}
    >
      <Card />
    </Col>
  </Row>
  <Row
    center="xs"
  >
    <Col>
      <Button
        primary={true}
      >
        Discover News
      </Button>
    </Col>
  </Row>
</DiscoverNews__DiscoverNewsContainer>
`;
